awslambdaric/bootstrap.py

```diff
@@ -367,9 +368,29 @@ def create_log_sink():
         return StandardLogSink()
 
 
+AWS_LAMBDA_LOG_FORMAT = os.environ.get("AWS_LAMBDA_LOG_FORMAT", "TEXT")
```

Comment:
We should also take into account the `AWS_LAMBDA_LOG_LEVEL` envvar ans set the logger verbosity accordingly